catroid/src/phiro/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -22,6 +22,26 @@
  */
 package org.catrobat.catroid.ui;
 
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+
+import org.catrobat.catroid.common.Constants;
+
 public class MainMenuActivity extends BaseMainMenuActivity {
-	//no adaptations for this flavor
+
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		enablePhiro();
+		super.onCreate(savedInstanceState);
+	}
+
+	private void enablePhiro() {
+		SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
+
+		if (!sharedPreferences.getBoolean(Constants.PHIRO_INITIALIZED, false)) {
+			BaseSettingsActivity.setPhiroSharedPreferenceEnabled(this, true);
```

Comment:
why is the same code still present in the SettingsActivity?

Reply:
True, that code should be removed.
I've removed it and also moved the Espresso test to a new PhiroMainMenuActivityTest-class.